Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -185,7 +183,6 @@ void EbrApplicationActivated(IActivatedEventArgs^ args) {
 
 void _ApplicationMainLaunch(ActivationType activationType, Platform::Object^ activationArg) {
     _ApplicationLaunch(activationType, activationArg);
-
     _appEvents = ref new AppEventListener();
     _appEvents->_RegisterEventHandlers();
```

Comment:
No, this requires the app to be created already but with the old activation that wouldn't be guaranteed to happen before we call these

---
In reply to: [72338254](https://github.com/Microsoft/WinObjC/pull/686#discussion_r72338254) [](ancestors = 72338254)